tests/app/services/test_process_request_json.py

```diff
@@ -58,3 +58,15 @@ def test_should_remove_raw_filter_fields_that_are_non_text():
     assert_equal(result["filter_lot"], "saas")
     assert_equal(result["filter_freeOption"], False)
     assert_equal("freeOption" in result, False)
+
+
+def test_should_add_parent_category():
+    request = {
+        "serviceCategories": ["Accounts payable"],
+    }
+
+    result = convert_request_json_into_index_json(request)
+    assert_equal(result["filter_serviceCategories"], [
+        "accountspayable",
+        "accountingandfinance",
```

Comment:
Value vs label is pretty confusing here, I'll try and explain:

 - the framework source data (i.e. the YAML) may have a value and/or a label for each option. Let's call these _source value_ and _source label_ respectively.
 - the indexing service gets passed the service data stored in the API. Let's call this the _API value_, and it comes from either _source value_ (if it's there, which it won't be in this case), or otherwise from _source label_. (Aside: it therefore might contain spaces and commas.)
 - the indexing service stores a transformation of what it gets passed, that doesn't contain spaces and commas. Let's call this the _Search value_.

If there's a _source value_, then the associated _source label_ is never passed into the API, so we can never index that. That's not the case here, but I have a feeling it's going to be an issue for other fields.

In this case, _API value_ == _source label_, and that does get indexed into `serviceCategories`. Existing code transforms that into a _Search value_, which goes in `filter_serviceCategories`. In other words, that's a squished version of _API value_, with commas and spaces removed.

Short answer, yes, we should be writing the labels into `serviceCategories`, I just didn't write a test for that.

I guess I copy-pasta'd the assert_equal, will fix.